contrib/cell_star/utils/cluster_util.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+__author__ = 'Adam Kaczmarek, Filip Mróz'
+
+# External imports
+import numpy as np
+# Internal imports
+from calc_util import euclidean_norm
+from contrib.cell_star.core.seed import Seed
+
+
```

Comment:
pdist is too slow. You can do the cross-product distance like this

    def pdist(list_points, _):
          pts = np.asanyarray(list_points)
          return np.sqrt(np.sum(pts[:, np.newaxis, :] - pts[np.newaxis, :, :], 2))
        return